rooms/rm_menu/rm_menu.yy

```diff
@@ -12,11 +12,14 @@
     {"name":"inst_4BA4DC55","path":"rooms/rm_menu/rm_menu.yy",},
     {"name":"inst_323BD67","path":"rooms/rm_menu/rm_menu.yy",},
     {"name":"inst_6AD60E5","path":"rooms/rm_menu/rm_menu.yy",},
+    {"name":"inst_66B4F716","path":"rooms/rm_menu/rm_menu.yy",},
+    {"name":"inst_672AF337","path":"rooms/rm_menu/rm_menu.yy",},
```

Comment:
I don't think we need two instances of the pause object. I think just one instance in the Instance layer will do.

Reply:
Removed second instance